Source/WebCore/animation/AnimationEffectTiming.cpp

```diff
@@ -95,7 +95,8 @@ BasicEffectTiming AnimationEffectTiming::getBasicTiming(const ResolutionData& da
             // Set unlimited current time based on the first matching condition:
             // - start time is resolved: (timeline time - start time) × playback rate
             // - Otherwise: animation's current time
-            ASSERT(data.timelineTime);
+            if (data.startTime)
+                ASSERT(data.timelineTime);
```

Comment:
ASSERT_IMPLIES?

Reply:
Ooh, fancy! Did not know about this. Thanks!